test/cpp/interop/xds_stats_watcher.cc

```diff
@@ -50,19 +55,32 @@ void XdsStatsWatcher::RpcCompleted(const AsyncClientCallResult& call,
       ++num_rpcs;
       auto rpcs_started = method_stat.rpcs_started();
       method_stat.set_rpcs_started(++rpcs_started);
+      auto rpc_metadata = metadata_by_peer_[peer].add_rpc_metadata();
```

Comment:
This code (lines 58 thru 67) gets executed even when peer is empty (line 38 is true). We should protect it when peer is empty, is it not?

Reply:
Done.